pagopa/io-app
# chore(IT Wallet): [SIW-954] Add new wallet base UI components

## Short description
This PR implements the first iteration of new wallet section base components

## List of changes proposed in this pull request
- Added DS section for **Wallet** components
- Added `withWalletCardBaseComponent` HOC and `WalletCardBaseComponent` type to create wallet cards components
- Adapted `IdPayCard` and `PaymentCard` components for the wallet
- Added `WalletCardCategoryContainer` component to handle card stacks grouped by category

## How to test
Go to **Profile > Design System > Wallet**, check that cards and category sections are displayed correctly

## Preview
<img width="200" alt="Screenshot 2024-03-22 alle 17 23 19" src="https://github.com/pagopa/io-app/assets/6160324/04e0393a-0e9e-45ec-9aa0-abd3ac8c8289">
